instrumentation/aerospike-client/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,29 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("com.aerospike")
+    module.set("aerospike-client")
+    versions.set("[4.0.0,)")
+    assertInverse.set(true)
+  }
+}
+
+dependencies {
+  library("com.aerospike:aerospike-client:8.0.0")
```

Comment:
What version do you want to support? Here you use 8, but in package name you use 7. Muzzle passes for 4, would that work?
Since the latest version seems to require java 21 you need to do something like
https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/c6a6fa622465ebf931ed5860987cb0ecc6a04a16/instrumentation/spring/spring-kafka-2.7/javaagent/build.gradle.kts#L83-L87